README.md

```diff
@@ -6,5 +6,16 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework that is written in Ruby that provides an organized approach to
+processing requests to a website backend
```

Comment:
And additionally, it makes it easy to spin up new applications very quickly because it starts us off with a lot of structure.